src/main/java/nextstep/sessions/domain/SessionStudent.java

```diff
@@ -45,4 +60,12 @@ public boolean equals(Object obj) {
     public int hashCode() {
         return Objects.hash(user);
     }
+
+    public void approval() {
```

Comment:
```suggestion
    public void approve() {
```
메서드 이름을 동사로 구현하는 것은 어떨까?